book/zh-cn/03-runtime.md

```diff
@@ -130,14 +130,14 @@ add(1.1, 2.2);
 
 ### `std::function`
 
-Lambda 表达式的本质是一个和函数对象类型相似的类类型（称为闭包类型）的对象（称为闭包对象），
+Lambda 表达式的本质是一个和函数对象类型相似的类型（称为闭包类型）的对象（称为闭包对象），
 当 Lambda 表达式的捕获列表为空时，闭包对象还能够转换为函数指针值进行传递，例如：
 
 ```cpp
 #include <iostream>
 
 using foo = void(int); // 定义函数类型, using 的使用见上一节中的别名语法
-void functional(foo f) { // 定义在参数列表中的函数类型 foo 被视为退化后的函数指针类型 foo*
+void func(foo f) { // 定义在参数列表中的函数类型 foo 被视为退化后的函数指针类型 foo*
```

Comment:
Why are you changing this? Did you see the code in line 148?

Reply:
Sorry, I use the "git add -p" command to stage code, make the mistake. the original meaning is to change "functional" to another word, for example, "func".

Reply:
Thanks for the clarification, could you please also change the corresponding english content? See [CONTRIBUTING](https://github.com/changkun/modern-cpp-tutorial/blob/master/CONTRIBUTING.md) for the details.